src/lumigo_tracer/event/event_trigger.py

```diff
@@ -133,10 +135,25 @@ def _is_event_bridge(event: dict):
     )
 
 
+def _is_appsync(event: dict) -> bool:
+    host = safe_get(event, ["context", "request", "headers", "host"])
+    return host and isinstance(host, str) and "appsync-api" in host
+
+
 def _parse_event_bridge(event: dict):
     return {"triggeredBy": "eventBridge", "messageId": event["id"]}
 
 
+def _parse_appsync(event: dict) -> dict:
+    headers = safe_get(event, ["context", "request", "headers"])
+    host = headers.get("host")
+    api_id = host.split(".")[0] if host and isinstance(host, str) else None
```

Comment:
```suggestion
    api_id = host.split(".")[0]
```
You can assume it :)

Reply:
can I assume it's a string?

Reply:
I think - you checked it in `_is_appsync`...

Reply:
oh yeah, but code pieces could change for some reason

Reply:
👍